### PR TITLE
Adding VFN port to haproxy.yaml

### PR DESCRIPTION
**Pull request by Markuze**
- [forge] schedule haproxy on validator nodegroup
- REMOVE THIS COMMIT
- Revert "REMOVE THIS COMMIT"
- [terraform] Adding VFN port to haproxy.yaml

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4670)

### CODE REVIEW

File: testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml [hunk 1]
```diff
@@ -32,5 +32,12 @@ service:
     enableRestApi: true
     enableMetricsPort: true
 
+# to give HAProxy access to more resources, schedule it on validators nodegroup
+# in the long term, we should consider giving the "utilities" nodegroup more resources
+# or removing it from the reference implementation entirely
+haproxy:
```
Comment (sitalkedia) [inline testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml:38]: This will be pretty expensive for us - essentially running haproxy on 16 vcore machine. Why is this needed? `haproxy` was hardly using any resources last time I checked.

Comment (Markuze) [reply testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml:38]: This was an experiment. These changes are not strictly needed.
